run.ps1

```diff
@@ -127,6 +127,6 @@ Add-Content $Env:GITHUB_PATH "$pwd\php-sdk\msys2\usr\bin"
 Add-Content $Env:GITHUB_PATH "$pwd\php-bin"
 Add-Content $Env:GITHUB_PATH "$pwd\php-dev"
 
-Write-Output "::set-output name=toolset::$toolset"
-Write-Output "::set-output name=prefix::$pwd\php-bin"
-Write-Output "::set-output name=vs::$vs"
+Write-Output "toolset=$toolset" >> $Env:GITHUB_OUTPUT
+Write-Output "prefix=$pwd\php-bin" >> $Env:GITHUB_OUTPUT
+Write-Output "vs=$vs" >> $Env:GITHUB_OUTPUT
```

Comment:
Just tested this PR on my fork of the extension and it looks good. See: https://github.com/jmikola/mongo-php-driver/actions/runs/3530605178

The builds were successful and no longer emit deprecation notices. The logs you see there are from a separate action we're using, which I've yet to patch.